src/test/java/org/apache/maven/plugins/pmd/CpdViolationCheckMojoTest.java

```diff
@@ -29,20 +32,16 @@ public class CpdViolationCheckMojoTest extends AbstractPmdReportTestCase {
     public void testDefaultConfiguration() throws Exception {
         generateReport("cpd", "default-configuration/cpd-default-configuration-plugin-config.xml");
 
-        // clear the output from previous pmd:cpd execution
-        CapturingPrintStream.init(true);
-
         try {
             File testPom = new File(
                     getBasedir(),
-                    "src/test/resources/unit/default-configuration/pmd-check-default-configuration-plugin-config.xml");
-            final CpdViolationCheckMojo cpdViolationMojo = (CpdViolationCheckMojo) lookupMojo(getGoal(), testPom);
-            cpdViolationMojo.execute();
+                    "src/test/resources/unit/default-configuration/cpd-check-default-configuration-plugin-config.xml");
+            final CpdViolationCheckMojo cpdViolationCheckMojo = (CpdViolationCheckMojo) lookupMojo(getGoal(), testPom);
+            cpdViolationCheckMojo.execute();
 
             fail("MojoFailureException should be thrown.");
-        } catch (final Exception e) {
-            assertTrue(e.getMessage()
-                    .startsWith("CPD " + AbstractPmdReport.getPmdVersion() + " has found 1 duplication."));
+        } catch (final MojoFailureException e) {
```

Comment:
There are asserts for expections, see https://junit.org/junit5/docs/current/user-guide/#writing-tests-exceptions-expected